lib/vanilla-masker.js

```diff
@@ -149,6 +151,10 @@
     }
     masked = masked.replace(clearDelimiter, "");
     masked = masked.length ? masked : "0";
+    var signal = "";
+    if(opts.showSignal === true) {
+      signal = value < 0 || value.startsWith('-') ? "-" : (opts.showPositiveSignal === true ? "+" : "");
```

Comment:
> How about two options: showNegativeSignal and showPositiveSignal, both defaulted to false?
Please no 😨 

It seems that @theodo is very interested in having this PR merged 😛.
Let's wait until tomorrow unless someone has an objection, and I'll press the green button.
For now, just remove the `showPositiveSignal` stuff, keep it simple.